ext/CMakeLists.txt

```diff
@@ -1,14 +1,16 @@
 set(ARMIPS_REGEXP OFF CACHE BOOL "" FORCE)
 
-add_subdirectory(armips)
-if(NOT USING_GLES2)
-  add_subdirectory(glew)
-endif()
+if(NOT WIIU)
+   add_subdirectory(armips)
```

Comment:
Would be nice to have armips because the new debugger (#10909) is going to link with it.  Might be nice to have that especially when it gets GE debugger capabilities.

-[Unknown]

Reply:
I only took it out temporarily because of some errors during compilation, which might take some time to fix. I still plan to bring that back in